src/battle-tooltips.ts

```diff
@@ -530,6 +530,7 @@ class BattleTooltips {
 
 		let zEffect = '';
 		let foeActive = pokemon.side.foe.active;
+		let allyActive = pokemon.side.active;
```

Comment:
```suggestion
		if (this.battle.gameType === 'freeforall') {
			foeActive = [...foeActive, ...pokemon.side.active].filter(active => active !== pokemon);
		}
```

I believe this is all you need, and then the other change can be reverted.